src/chainer.js

```diff
@@ -8,11 +10,12 @@ function chainer(functions) {
     let result = x;
 
```

Comment:
The variable 'x' is not defined in the function. If it's a parameter of the function, include it in the function declaration. It should be 'function chainer(functions, x) {...}'.